src/condor_utils/xform_utils.cpp

```diff
@@ -46,8 +46,10 @@
 #include <set>
 
 // values for hashtable defaults, these are declared as char rather than as const char to make g++ on fedora shut up.
-static char OneString[] = "1", ZeroString[] = "0";
-//static char ParallelNodeString[] = "#pArAlLeLnOdE#";
+#if defined(WIN32) || defined(LINUX)
+static char OneString[] = "1"
```

Comment:
There is a missing semicolon here.
```suggestion
static char OneString[] = "1";
```